test/ticket.js

```diff
@@ -0,0 +1,17 @@
+const TicketClass = require('../src/TicketClass.js')
+const chai = require('chai');
+const expect = chai.expect;
+const bookshelf = require('../database/connection.js')
+var chaiAsPromised = require("chai-as-promised");
+chai.use(chaiAsPromised);
+
+const myTicket = new TicketClass()
+
+describe ('Making database calls on Ticket ', () =>{
+  context('Fetching Tickets ', () =>{
+    it('Tickets do not exist yet!', () =>{
+      expect(true).to.be.true
```

Comment:
Yeah, we currently have no data in the tickets table.  We need to add functions for Passenger to buy a ticket to generate one, and added this test in the meantime just to put something on the page. 

Is it better practice to leave this file empty if we're not testing anything yet?